app/modules/asset_groups/models.py

```diff
@@ -128,12 +110,17 @@ class AssetGroupSighting(db.Model, HoustonModel):
     )
     asset_group = db.relationship('AssetGroup', backref=db.backref('sightings'))
 
-    meta = db.Column(db.JSON, nullable=True)
+    # configuration metadata from the create request
+    config = db.Column(db.JSON, nullable=True)
+
+    # May have multiple jobs outstanding, store as json array of job_ids
+    jobs = db.Column(db.JSON, nullable=True)
```

Comment:
If we do make a broader Job or Task class for many houston initiated IA actions it would make sense to leverage it here. It would probably look similar to the AssetGroupJob, but would pack more variable information into a JSON field. That would be a separate PR of course and this simplification is good for now.

Reply:
Have a look at what I put for this in the IA Pipeline doc. I don't think there will need to be much more here.